Button.js

```diff
@@ -5,6 +5,7 @@ import {
     TouchableHighlight,
     View,
 } from 'react-native';
+import PropTypes from 'prop-types'
```

Comment:
Would you mind adding a `;` at the end to keep up with the linting rules please?